.github/workflows/reusable-build-upload.yaml

```diff
@@ -33,6 +33,12 @@ on:
         required: true
         type: string
 
+      github-release:
+        description: "Whether to upload the build as a github release"
+        required: false
+        default: "true"
```

Comment:
Just wondering, why is it not required but defaulted to true?

Reply:
Mentioned it above, but, for regular releases we /do/ want to run a github release, so I had the default set to true so that I didn't have to change `release.yaml`.